python/cuml/test/test_pickle.py

```diff
@@ -70,13 +70,19 @@
     'AutoARIMA',
     'KalmanFilter',
     'BaseRandomForestModel',
-    'ForestInference'
+    'ForestInference',
+    'MulticlassClassifier',
+    'OneVsOneClassifier',
+    'OneVsRestClassifier'
 ]
 unfit_clone_xfail = [
     'AutoARIMA',
     "ARIMA",
     "BaseRandomForestModel",
     "GaussianRandomProjection",
+    'MulticlassClassifier',
+    'OneVsOneClassifier',
+    'OneVsRestClassifier',
```

Comment:
Why do these fail to pickle and clone?

Reply:
Can you elaborate more on this? I thought `clone()` effectively did the following:
```python
def clone(old_est):
    old_params = old_est.get_params()
    new_est = old_est.__class__(**old_params)
    return new_est
```
As long as `estimator` is returned by `get_params()`, it should work, right?

Reply:
Cloning itself does work. But the cloning test creates the estimator without any params:
https://github.com/rapidsai/cuml/blob/a0f45496169bffe485a825be7152e82de12e90cb/python/cuml/test/test_pickle.py#L366

This would not work without default value for the `estimator` arg.

Btw, testing cloning revealed a bug in returning the param names, I have corrected it in by 54896e59a.